src/lib.rs

```diff
@@ -49,21 +49,21 @@ use std::marker::PhantomData;
 use std::ops::{Deref, DerefMut};
 use std::path::{Path, PathBuf};
 use std::rc::Rc;
-use std::sync::mpsc::{Receiver, channel};
+use std::sync::mpsc::{channel, Receiver};
 use std::time::{Duration, Instant};
 
 /// Loadable object from either the file system or memory.
 ///
 /// An object can be loaded if it can output a `Loaded<_>`. It’s important to note that you’re not
 /// supposed to use that trait directly. Instead, you should use the `Store`’s functions.
-pub trait Load: 'static + Sized {
+pub trait Load<C>: 'static + Sized {
   /// Type of the key used to load the resource.
   ///
   /// You have two choices:
   ///
   /// - `PathKey`, used to load your resource from the file system.
   /// - `LogicalKey`, used to compute your resource from memory.
-  type Key: Into<DepKey>;
+  type Key: Into<DepKey> + 'static;
```

Comment:
Why is that needed?

Reply:
As far as I got, it's needed "because the compiler complains if it isn't there".  :-P  I'm still playing with it.